mongodb_update_block.py

```diff
@@ -27,7 +27,7 @@ class MongoDBUpdate(MongoDBBase):
         multi (str): Perform update on only one matche if False.
 
     """
-    spec = Property(title='Query Document', default="{'id': {{$id}} }")
+    spec = Property(title='Query Document', default="{{ {'_id': $id } }}")
```

Comment:
Just to clarify the issue that we're trying to fix, it looks like the expression is actually allowed to be a string representation of a dict. Inside `MongoDBUpdate.execute_query` we call `evaluate_expression`, which is defined in the base class https://github.com/nio-blocks/mongo/blob/master/mongodb_base.py#L173 , to get a dictionary from the string.

I tried reproducing what I understood to be your issue from our conversation earlier, but have not been successful so far. What happens if you use `{'_id': {{$id}} }` for your Query Document?

Reply:
When using ```{'_id': {{$id}} }```, I get an error in the logger:
```
[2018-11-12T22:06:08.555708Z][ERROR][MongoDB.Update with Ack] Query failed
```

Reply:
That sure looks like it should have worked. We'll have to look at the traceback to see why it didn't work, can you please SSH to your device and open `<nio-project>logs/MongoDB.log`, look for `Query failed`, and paste here the entire log entry?

And just FYI, if you update your binary to the latest the logger panel will include these tracebacks https://account.n.io/binaries/download

Reply:
Can you please verify the version of the framework and binary that you have running for me when you can? Please run: `pip3 list | grep nio`

The newest framework is nio 3.2.1, and the newest binary is 20181026.

Reply:
https://gist.github.com/sploders101/4769886a5a33ce0936d42672f97a8747